spec/index.bs

```diff
@@ -4315,6 +4315,7 @@ GPUQueue includes GPUObjectBase;
         The operation throws {{OperationError}} if any of the following any of the following requirements are unmet:
 
         - {{GPUQueue/copyImageBitmapToTexture(source, destination, copySize)/copySize}}.[=Extent3D/depth=] must be `1`.
+        - {{GPUQueue/copyImageBitmapToTexture(source, destination, copySize)/destination}}.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}} must be {{GPUTextureFormat/"rgba8unorm"}} or {{GPUTextureFormat/"rgba8unorm-srgb"}} or {{GPUTextureFormat/"bgra8unorm"}} or {{GPUTextureFormat/"bgra8unorm-srgb"}} or {{GPUTextureFormat/"rgb10a2unorm"}} or {{GPUTextureFormat/"rgba16float"}} or {{GPUTextureFormat/"rgba32float"}} or {{GPUTextureFormat/"rg8unorm"}} or {{GPUTextureFormat/"rg16float"}}.
```

Comment:
```suggestion
        - {{GPUQueue/copyImageBitmapToTexture(source, destination, copySize)/destination}}.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}} must be one of the following:
             - {{GPUTextureFormat/"rgba8unorm"}} 
             - {{GPUTextureFormat/"rgba8unorm-srgb"}}
             - {{GPUTextureFormat/"bgra8unorm"}}
             - {{GPUTextureFormat/"bgra8unorm-srgb"}}
             - {{GPUTextureFormat/"rgb10a2unorm"}}
             - {{GPUTextureFormat/"rgba16float"}}
             - {{GPUTextureFormat/"rgba32float"}}
             - {{GPUTextureFormat/"rg8unorm"}}
             - {{GPUTextureFormat/"rg16float"}}
```

Reply:
Why are RG formats listed here? It seems that ImageBitmap always have 4 channels (or 3) so it might not be useful to copy them to 2-channel formats.

Reply:
I think an ImageBitmap could also have 1 channel.

But leaving this list more restrictive for now is probably best.

Reply:
WebGL 2.0 does allow uploads into 2-channel formats: https://www.khronos.org/registry/webgl/specs/latest/2.0/#3.7.6

and in fact it's possible for some image sources (not sure about ImageBitmap) to be 2-channel: grayscale-alpha.

And I think it's reasonable to support them. Implementation cost should be fairly low and they definitely have value to some authors (e.g. 2-channel normal map stored in a PNG, which is not too uncommon).

Reply:
Thanks @kainino0x for the inputs. And yes, I put this because:
1. It's summarized from the implementation which skia support the conversion.
2. In the discussion of WebCodec in our team, we find two channel and one channel formats(missed because implementation doesn't support it yet) may also be needed.
But I think Kai's input are more solid.